src/platform/packages/shared/shared-ux/chrome/navigation/src/ui/components/feedback_btn.tsx

```diff
@@ -14,6 +14,7 @@ import type { SolutionId } from '@kbn/core-chrome-browser';
 
 const feedbackUrls: { [id in SolutionId]: string } = {
   es: 'https://ela.st/search-nav-feedback',
+  chat: 'https://ela.st/search-nav-feedback',
```

Comment:
I used the `search` url for now given we don't have an equivalent for workchat today. We'll update it when we figure that out